roles/copy_cert_files/tasks/main.yml

```diff
@@ -45,3 +45,23 @@
     dest: "{{ item.dest }}"
     state: link
   loop: "{{ migrid_cert_file_links }}"
+
+# copy selfsigned root certificate to external-certificates/
+
+- name: Create directory for external-certificates/ if needed
+  file:
+    path: "{{ migrid_root }}/external-certificates"
+    owner: "{{ migrid_adm }}"
+    group: "{{ migrid_adm }}"
+    state: directory
+    mode: "0755"
+  when: migrid_docker_compose_override==true
```

Comment:
What has `migrid_docker_compose_override` to do with it? shouldn't this be `migrid_selfsigned_certificates`?

To reduce the amount of vars it would also be possible to check:

```
when: migrid_selfsigned_certificates is defined
```